internal/scheduleringester/config.go

```diff
@@ -13,6 +13,8 @@ type Configuration struct {
 	Metrics configuration.MetricsConfig
 	// General Pulsar configuration
 	Pulsar configuration.PulsarConfig
+	// Map of allowed priority classes to priority
```

Comment:
Not sure I really understand what this comment is trying to say.

Reply:
I'll change it to `Map of allowed priority classes name to priority value`